packages/components/src/Page/Page.tsx

```diff
@@ -32,21 +34,51 @@ interface PageProps {
    * @default standard
    */
   readonly width?: "fill" | "standard" | "narrow";
+
+  /**
+   * Page title primary action button settings.
+   */
+  readonly primaryAction?: ButtonProps;
+
+  /**
+   * Page title secondary action button settings.
+   */
+  readonly secondaryAction?: ButtonProps;
+
+  /**
+   * Page title Action menu.
+   */
+  readonly moreActionsMenu?: SectionProps[];
 }
 
 export function Page({
   title,
   intro,
   children,
   width = "standard",
+  primaryAction,
+  secondaryAction,
+  moreActionsMenu = [],
 }: PageProps) {
-  const className = classnames(styles.page, styles[width]);
+  const pageStyles = classnames(styles.page, styles[width]);
+
+  const showMenu = moreActionsMenu.length > 0;
+  const showActionGroup = showMenu || primaryAction;
```

Comment:
```suggestion
  const showActionGroup = showMenu || primaryAction || secondaryAction;
```

Reply:
That makes sense to me

Reply:
As discussed would be good to add a note to the props to this effect.